api-service/src/controllers/DatasetList/DatasetList.ts

```diff
@@ -12,7 +12,7 @@ export const apiId = "api.datasets.list"
 export const errorCode = "DATASET_LIST_FAILURE"
 const liveDatasetStatus = ["Live", "Retired", "Purged"]
 const draftDatasetStatus = ["Draft", "ReadyToPublish"]
-const defaultFields = ["dataset_id", "name", "type", "status", "tags", "version", "api_version", "dataset_config", "created_date", "updated_date"]
+const defaultFields = ["dataset_id", "name", "type", "status", "tags", "version", "api_version", "dataset_config", "created_date", "updated_date","data_schema"]
```

Comment:
Why do we need data_schema as the default fields?

Reply:
currently im using it in web-console 
i'll remove after implementing the fields api